[DROOLS-6063] drools-decisiontables takes "100%" as String even in Pe… (#3425)
kiegroup/kie-docs: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-rule-set-entries-ref.adoc

|`true` or `false`. If `true`, then quotation marks are escaped so that they appear literally in the DRL.
|Optional, at most once. If omitted, quotation marks are escaped.

|`IgnoreNumericFormat`

> |`true` or `false`. If `true`, then the format for numeric values will be ignored (e.g. `Percent`, `Currency`).

Reviewer: ```suggestion
|`true` or `false`. If `true`, then the format for numeric values is ignored, for example, percent and currency.
```

Author: Thanks @hmanwani-rh , applied the suggestion.
